lib/datasets/version.rb

```diff
@@ -1,3 +1,3 @@
 module Datasets
-  VERSION = "0.1.5"
+  VERSION = "0.1.6"
```

Comment:
Could you remove this change?
We'll do this AFTER we release `0.1.5`.

Reply:
fix: [34316f1](https://github.com/red-data-tools/red-datasets/pull/156/commits/34316f166c340bf6c91175a9af14a8b78ee5189e): I'm sorry I misunderstood it.